lib/commanded/subscriptions/subscriptions.ex

```diff
@@ -119,30 +130,30 @@ defmodule Commanded.Subscriptions do
     {:reply, :ok, state}
   end
 
-  def handle_call({:register_subscription, name, pid}, _from, %Subscriptions{subscriptions_table: subscriptions_table} = state) do
+  def handle_info({:register_subscription, name, pid}, %Subscriptions{subscriptions_table: subscriptions_table} = state) do
     :ets.insert(subscriptions_table, {name, pid})
 
-    {:reply, :ok, state}
+    {:noreply, state}
   end
 
-  def handle_cast({:ack_event, name, stream_uuid, stream_version, pid}, %Subscriptions{streams_table: streams_table, subscriptions_table: subscriptions_table, started_at: started_at} = state) do
-    # track insert date as ms since the subscripions process was started to support expiry of stale acks
+  def handle_info({:ack_event, name, stream_uuid, stream_version, pid}, %Subscriptions{streams_table: streams_table, subscriptions_table: subscriptions_table, started_at: started_at} = state) do
+    # track insert date as milliseconds since the subscriptions process was
+    # started to support expiry of stale acks
     inserted_at_epoch = NaiveDateTime.diff(now(), started_at, :second)
```

Comment:
Is it normal for the comment to say "track (...) as milliseconds", and then the `diff` function is called with unit ``:second`?

Reply:
No, the comment is a lie! I'll fix it, thanks.